installscripts/dockerfiles/jenkins-ce/Dockerfile

```diff
@@ -5,9 +5,13 @@ MAINTAINER JazzOSS Team
 # Switching to root to configure the image with system packages
 USER root
 RUN apt-get update && apt-get install -y vim curl sudo libtool autoconf make unzip rsync gcc autogen shtool pkg-config lsb-release python python-dev python-pip python-setuptools groff less && \
-curl https://bootstrap.pypa.io/get-pip.py | python && pip install --upgrade awscli && apt-get clean && pip install virtualenv && /usr/bin/easy_install virtualenv
+python3.6 python3-pip build-essential libssl-dev libffi-dev python3-dev && curl https://bootstrap.pypa.io/get-pip.py | python && curl https://bootstrap.pypa.io/get-pip.py | python3 && pip install --upgrade awscli && \
+pip3 install --upgrade awscli  apt-get clean && pip install virtualenv && pip3 install virtualenv && /usr/bin/easy_install virtualenv
```

Comment:
is upgrade awscli needed?
also why is it in above line as well?

Reply:
It is not required and removed it.